cpmonitor/views.py

```diff
@@ -257,24 +256,30 @@ def cap_checklist_view(request, city_slug):
     return render(request, "cap_checklist.html", context)
 
 
-def _get_cap_checklist(city) -> list:
+def _get_cap_checklist(city) -> dict:
     try:
-        checklist_items = city.cap_checklist._meta.get_fields()
+        checklist = city.cap_checklist
     except CapChecklist.DoesNotExist:
         return {}
 
-    checklist_items = [
-        item for item in checklist_items if item.attname not in ["city_id", "id"]
+    return _as_formatted_checklist(checklist)
+
+
+def _as_formatted_checklist(checklist):
+    checkbox_items = [
+        field
+        for field in checklist._meta.get_fields()
+        if field.attname not in ["city_id", "id"] and "_rationale" not in field.attname
     ]
 
-    return [
-        {
-            "question": item.verbose_name,
-            "check": getattr(city.cap_checklist, item.attname),
-            "help_text": item.help_text,
+    return {
+        checkbox_item.verbose_name: {
```

Comment:
I did like the previous structure a little better, but that is due to the conflict merge, I guess.